koku/masu/external/downloader/oci/oci_report_downloader.py

```diff
@@ -403,12 +412,16 @@ def download_file(self, key, stored_etag=None, manifest_id=None, start_date=None
                 manifest_id,
                 self.context,
             )
+
+            if not file_names and not date_range:
+                raise OCIReportDownloaderError("Empty report retrieved")
+
         except Exception as err:
             err_msg = (
-                "Could not complete download."
-                f"\n  Provider: {self._provider_uuid}"
-                f"\n  Customer: {self.customer_name}"
-                f"\n  Response: {err.message}"
+                "Could not complete download. "
+                f"Provider: {self._provider_uuid}. "
+                f"Customer: {self.customer_name}. "
+                f"Response: {err}."
```

Comment:
Does it make sense to remove the line breaks here? We do this in a few logged places to make it easier to read/find.